src/javascript/app_2/components/elements/drawer/index.jsx

```diff
@@ -0,0 +1,6 @@
+export * from './drawer_header.jsx';
+// export * from './portfolio_drawer.jsx';
```

Comment:
`portfolio_drawer` is unrelated to drawer components. Probably better to keep it out of this folder and remove this comment.